src/monitor-pipeline-group/azext_monitor_pipeline_group/aaz/latest/monitor/pipeline_group/_create.py

```diff
@@ -0,0 +1,1157 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "monitor pipeline-group create",
+)
+class Create(AAZCommand):
+    """Create a pipeline group instance.
+    """
```

Comment:
please add example for create and update cmd

Reply:
Added example.